crates/router/src/connector/payme/transformers.rs

```diff
@@ -1031,13 +1048,24 @@ impl<F, T>
             .items
             .first()
             .ok_or(errors::ConnectorError::ResponseHandlingFailed)?;
-        Ok(Self {
-            response: Ok(types::RefundsResponseData {
-                refund_status: enums::RefundStatus::try_from(
-                    pay_sale_response.sale_status.clone(),
-                )?,
+        let refund_status = enums::RefundStatus::try_from(pay_sale_response.sale_status.clone())?;
+        let response = if is_refund_failure(refund_status) {
+            Err(types::ErrorResponse {
+                code: consts::NO_ERROR_CODE.to_string(),
```

Comment:
Don't we get error code, message, during retrieve call?

Reply:
Couldn't find any such fields in the API documentation: https://docs.payme.io/docs/payments/13919edc16077-list-sales#Responses